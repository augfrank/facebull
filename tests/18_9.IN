M0   C0   C1  2709
M1   C0   C2  2816
M2   C0   C3  2046
M3   C0   C4  2990
M4   C0   C5  2489
M5   C0   C6  2732
M6   C0   C7  2560
M7   C0   C8  2566
M8   C0   C9  2006
M9   C0   C10  2736
M10   C0   C11  2416
M11   C0   C12  2805
M12   C0   C13  2484
M13   C0   C14  2557
M14   C0   C15  2913
M15   C0   C16  2419
M16   C0   C17  2822
M17   C1   C0  2641
M18   C1   C2  2523
M19   C1   C3  2043
M20   C1   C4  2680
M21   C1   C5  2043
M22   C1   C6  2191
M23   C1   C7  2896
M24   C1   C8  2331
M25   C1   C9  2645
M26   C1   C10  2873
M27   C1   C11  2273
M28   C1   C12  2896
M29   C1   C13  2589
M30   C1   C14  2017
M31   C1   C15  2869
M32   C1   C16  2014
M33   C1   C17  2684
M34   C2   C0  2350
M35   C2   C1  2606
M36   C2   C3  2424
M37   C2   C4  2622
M38   C2   C5  2181
M39   C2   C6  2750
M40   C2   C7  2284
M41   C2   C8  2510
M42   C2   C9  2442
M43   C2   C10  2601
M44   C2   C11  2680
M45   C2   C12  2572
M46   C2   C13  2321
M47   C2   C14  2247
M48   C2   C15  2919
M49   C2   C16  2596
M50   C2   C17  2480
M51   C3   C0  2696
M52   C3   C1  2248
M53   C3   C2  2260
M54   C3   C4  2400
M55   C3   C5  2518
M56   C3   C6  2833
M57   C3   C7  2409
M58   C3   C8  2861
M59   C3   C9  2106
M60   C3   C10  2921
M61   C3   C11  2093
M62   C3   C12  2025
M63   C3   C13  2431
M64   C3   C14  2667
M65   C3   C15  2664
M66   C3   C16  2893
M67   C3   C17  2054
M68   C4   C0  2782
M69   C4   C1  2200
M70   C4   C2  2741
M71   C4   C3  2152
M72   C4   C5  2585
M73   C4   C6  2647
M74   C4   C7  2269
M75   C4   C8  2124
M76   C4   C9  2017
M77   C4   C10  2377
M78   C4   C11  2694
M79   C4   C12  2746
M80   C4   C13  2742
M81   C4   C14  2185
M82   C4   C15  2989
M83   C4   C16  2182
M84   C4   C17  2378
M85   C5   C0  2279
M86   C5   C1  2615
M87   C5   C2  2946
M88   C5   C3  2169
M89   C5   C4  2565
M90   C5   C6  2948
M91   C5   C7  2853
M92   C5   C8  2477
M93   C5   C9  2072
M94   C5   C10  2629
M95   C5   C11  2437
M96   C5   C12  2360
M97   C5   C13  2640
M98   C5   C14  2149
M99   C5   C15  2529
M100   C5   C16  2826
M101   C5   C17  2484
M102   C6   C0  2824
M103   C6   C1  2637
M104   C6   C2  2686
M105   C6   C3  2124
M106   C6   C4  2341
M107   C6   C5  2525
M108   C6   C7  2341
M109   C6   C8  2399
M110   C6   C9  2650
M111   C6   C10  2952
M112   C6   C11  2458
M113   C6   C12  2572
M114   C6   C13  2289
M115   C6   C14  2809
M116   C6   C15  2514
M117   C6   C16  2980
M118   C6   C17  2531
M119   C7   C0  2678
M120   C7   C1  2699
M121   C7   C2  2093
M122   C7   C3  2098
M123   C7   C4  2063
M124   C7   C5  2848
M125   C7   C6  2404
M126   C7   C8  2349
M127   C7   C9  2141
M128   C7   C10  2172
M129   C7   C11  2257
M130   C7   C12  2538
M131   C7   C13  2875
M132   C7   C14  2572
M133   C7   C15  2226
M134   C7   C16  2161
M135   C7   C17  2363
M136   C8   C0  2069
M137   C8   C1  2648
M138   C8   C2  2386
M139   C8   C3  2921
M140   C8   C4  2289
M141   C8   C5  2060
M142   C8   C6  2176
M143   C8   C7  2089
M144   C8   C9  2746
M145   C8   C10  2848
M146   C8   C11  2159
M147   C8   C12  2176
M148   C8   C13  2929
M149   C8   C14  2047
M150   C8   C15  2437
M151   C8   C16  2134
M152   C8   C17  2348
M153   C9   C0  2824
M154   C9   C1  2207
M155   C9   C2  2619
M156   C9   C3  2278
M157   C9   C4  2092
M158   C9   C5  2680
M159   C9   C6  2384
M160   C9   C7  2730
M161   C9   C8  2228
M162   C9   C10  2546
M163   C9   C11  2947
M164   C9   C12  2608
M165   C9   C13  2604
M166   C9   C14  2417
M167   C9   C15  2787
M168   C9   C16  2343
M169   C9   C17  2280
M170   C10   C0  2309
M171   C10   C1  2973
M172   C10   C2  2355
M173   C10   C3  2390
M174   C10   C4  2711
M175   C10   C5  2770
M176   C10   C6  2513
M177   C10   C7  2791
M178   C10   C8  2920
M179   C10   C9  2490
M180   C10   C11  2073
M181   C10   C12  2576
M182   C10   C13  2775
M183   C10   C14  2875
M184   C10   C15  2648
M185   C10   C16  2495
M186   C10   C17  2600
M187   C11   C0  2886
M188   C11   C1  2931
M189   C11   C2  2502
M190   C11   C3  2457
M191   C11   C4  2052
M192   C11   C5  2411
M193   C11   C6  2267
M194   C11   C7  2578
M195   C11   C8  2656
M196   C11   C9  2215
M197   C11   C10  2170
M198   C11   C12  2094
M199   C11   C13  2563
M200   C11   C14  2762
M201   C11   C15  2342
M202   C11   C16  2973
M203   C11   C17  2653
M204   C12   C0  2986
M205   C12   C1  2434
M206   C12   C2  2379
M207   C12   C3  2477
M208   C12   C4  2257
M209   C12   C5  2012
M210   C12   C6  2448
M211   C12   C7  2389
M212   C12   C8  2792
M213   C12   C9  2236
M214   C12   C10  2734
M215   C12   C11  2316
M216   C12   C13  2814
M217   C12   C14  2604
M218   C12   C15  2051
M219   C12   C16  2711
M220   C12   C17  2426
M221   C13   C0  2531
M222   C13   C1  2276
M223   C13   C2  2632
M224   C13   C3  2258
M225   C13   C4  2756
M226   C13   C5  2474
M227   C13   C6  2205
M228   C13   C7  2660
M229   C13   C8  2889
M230   C13   C9  2197
M231   C13   C10  2221
M232   C13   C11  2305
M233   C13   C12  2493
M234   C13   C14  2254
M235   C13   C15  2992
M236   C13   C16  2302
M237   C13   C17  2189
M238   C14   C0  2003
M239   C14   C1  2081
M240   C14   C2  2414
M241   C14   C3  2698
M242   C14   C4  2361
M243   C14   C5  2121
M244   C14   C6  2666
M245   C14   C7  2185
M246   C14   C8  2564
M247   C14   C9  2403
M248   C14   C10  2622
M249   C14   C11  2829
M250   C14   C12  2782
M251   C14   C13  2261
M252   C14   C15  2998
M253   C14   C16  2233
M254   C14   C17  2790
M255   C15   C0  2039
M256   C15   C1  2823
M257   C15   C2  2682
M258   C15   C3  2615
M259   C15   C4  2821
M260   C15   C5  2087
M261   C15   C6  2833
M262   C15   C7  2191
M263   C15   C8  2854
M264   C15   C9  2081
M265   C15   C10  2310
M266   C15   C11  2280
M267   C15   C12  2663
M268   C15   C13  2876
M269   C15   C14  2013
M270   C15   C16  2694
M271   C15   C17  2305
M272   C16   C0  2941
M273   C16   C1  2071
M274   C16   C2  2281
M275   C16   C3  2872
M276   C16   C4  2223
M277   C16   C5  2772
M278   C16   C6  2656
M279   C16   C7  2184
M280   C16   C8  2838
M281   C16   C9  2264
M282   C16   C10  2288
M283   C16   C11  2051
M284   C16   C12  2074
M285   C16   C13  2229
M286   C16   C14  2943
M287   C16   C15  2127
M288   C16   C17  2100
M289   C17   C0  2039
M290   C17   C1  2704
M291   C17   C2  2136
M292   C17   C3  2931
M293   C17   C4  2499
M294   C17   C5  2509
M295   C17   C6  2180
M296   C17   C7  2691
M297   C17   C8  2379
M298   C17   C9  2839
M299   C17   C10  2699
M300   C17   C11  2371
M301   C17   C12  2798
M302   C17   C13  2211
M303   C17   C14  2432
M304   C17   C15  2483
M305   C17   C16  2044
