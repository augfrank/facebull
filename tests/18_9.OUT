37836
2 32 38 67 75 93 114 121 143 154 180 191 218 231 238 269 284 295
